### DISCUSSION TITLE
Thank You!!

### DISCUSSION THREAD

**Discussion by daxkamala** [davesteele/comitup#133] [General]
Just want to offer my thanks for something that works.
Took me a long time to find your solution and I've tried and struggled with pretty much all the other offerings without success.
Like many, I want to be able to send systems out with an easy option to configure the local WiFi.
Yours is the first one that works reliably.
Next, I'm going to try comitup on a bananapi zero, that should be fun....

**Comment by davesteele** [comment]
Thanks for the feedback. I'd be curious to know what's involved with getting it to work on the bananapi.

**Reply by jacycatlin** [reply]
Speaking of porting, I'm hoping to get it working on PiKVM which is an Arch-based project.  But by "get it working" I mean "find someone who can do it" as my programmer days are decades in the past and I have little hope of working it out myself... I understand it's mainly Python so the main changes would be to how it sets the WiFi config?
Anyhow, this is an awesome project, if I'd known about it a few years ago I'd have put it into my Dashboard project, which never got off the ground 'cos I couldn't come up with a way for end users to set it up on their own WiFi...

**Comment by sistemicorp** [comment]
I also submit my Thanks!  I am just getting started with it after spending 3 days on other solutions.
I am using it on a RPi4B.

**Comment by WernerFettke** [comment]
Also just want to say Thanks !!! ComitUp Rocks !!!

**Reply by davesteele** [reply]
I appreciate the feedback.

**Comment by mennamorato** [comment]
I also just ran across this and am running it on a pi zero w.
Seems to work well and I added my python script to run after it gets to CONNECTED state, so my project is almost complete. I struggled with looking for ways to do this on a PI as well.. Thanks Dave!

**Reply by davesteele** [reply]
I'm glad it is useful for you.

Make sure you are on the most recent version.

**Reply by mennamorato** [reply]
v 1.24

**Reply by davesteele** [reply]
You should [upgrade](https://davesteele.github.io/comitup/ppa.html).

Just apt upgrade if you are using the comitup image.
